assets/js/src/BoardCollapsedMode.js

```diff
@@ -1,27 +1,57 @@
-Kanboard.BoardCollapsedMode = function(app) {
-    this.app = app;
-};
-
-Kanboard.BoardCollapsedMode.prototype.keyboardShortcuts = function() {
-    var self = this;
-
-    if (self.app.hasId("board")) {
-        Mousetrap.bind("s", function() {
-            self.toggle();
-        });
-    }
-};
-
-Kanboard.BoardCollapsedMode.prototype.toggle = function() {
-    var self = this;
-    this.app.showLoadingIcon();
-
-    $.ajax({
-        cache: false,
-        url: $('.filter-display-mode:not([style="display: none;"]) a').attr('href'),
-        success: function(data) {
-            $('.filter-display-mode').toggle();
-            self.app.get("BoardDragAndDrop").refresh(data);
-        }
-    });
-};
\ No newline at end of file
+Kanboard.BoardCollapsedMode = function(app) {
+    this.app = app;
+};
+
+Kanboard.BoardCollapsedMode.prototype.keyboardShortcuts = function() {
+    var self = this;
+
+    if (self.app.hasId("board-container")) {
+        Mousetrap.bind("s", function() {
+            self.toggle();
+        });
+    }
+};
+
+Kanboard.BoardCollapsedMode.prototype.toggle = function() {
+    var self = this;
+    this.app.showLoadingIcon();
+
+    var url = $('.filter-display-mode:not([style="display: none;"]) a').attr('href');
+
+    if (self.app.hasId("bigboard")) {

```

Comment:
The Kanboard's code should not include code related to your plugin. I agree to improve this code to make your plugin compatible, but not to hardcode the name of your plugin here.